specification/communication/Communication.JobRouter/routes.tsp

```diff
@@ -0,0 +1,366 @@
+import "@azure-tools/typespec-azure-core";
+import "@typespec/http";
+import "@typespec/rest";
+import "./models.tsp";
+
+using TypeSpec.Rest;
+using TypeSpec.Http;
+using Azure.Core.Traits;
+
+namespace AzureCommunicationRoutingService;
+
+alias ServiceTraits = NoRepeatableRequests &
+  NoConditionalRequests & NoClientRequestId;
+alias PatchOperationTraits = NoRepeatableRequests & NoClientRequestId & SupportsRouterConditionalPatchUpdateRequests & RouterReturnConditionalResponseHeaders;
+alias GetOperationTraits = ServiceTraits & RouterReturnConditionalResponseHeaders;
+
+alias Operations = Azure.Core.ResourceOperations<ServiceTraits>;
+alias GetOperations = Azure.Core.ResourceOperations<GetOperationTraits>;
+alias PatchOperations = Azure.Core.ResourceOperations<PatchOperationTraits>;
+
+interface JobRouterAdministrationOperations {
+  #suppress "@azure-tools/typespec-azure-core/no-explicit-routes-resource-ops" "Codegen cannot import prefixes from namespace"
+  @summary("Creates or updates a classification policy.")
+  @doc("Creates or updates a classification policy.")
+  @route("/routing")
+  upsertClassificationPolicy is PatchOperations.ResourceCreateOrUpdate<ClassificationPolicy>;
+  
+  #suppress "@azure-tools/typespec-azure-core/no-explicit-routes-resource-ops" "Codegen cannot import prefixes from namespace"
+  @summary("Retrieves an existing classification policy by Id.")
+  @doc("Retrieves an existing classification policy by Id.")
+  @route("/routing")
+  getClassificationPolicy is GetOperations.ResourceRead<ClassificationPolicy>;
+  
+  #suppress "@azure-tools/typespec-azure-core/no-explicit-routes-resource-ops" "Codegen cannot import prefixes from namespace"
+  @summary("Delete a classification policy by Id.")
+  @doc("Delete a classification policy by Id.")
+  @route("/routing")
+  deleteClassificationPolicy is Operations.ResourceDelete<ClassificationPolicy>;
+  
+  #suppress "@azure-tools/typespec-azure-core/use-standard-operations"
```

Comment:
Please explain why you can't use an Azure Core operation template here.

Reply:
From what it looks like here, "routing" is a root singleton resource. And I don't think we support (or necessarily want to support ) `/` as a resource collection.

Reply:
More context here [Teams post internal](https://teams.microsoft.com/l/message/19:906c1efbbec54dc8949ac736633e6bdf@thread.skype/1698882012944?tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47&groupId=3e17dcb0-4257-4a30-b843-77f47f1d4121&parentMessageId=1698882012944&teamName=Azure%20SDK&channelName=TypeSpec%20Discussion%20%F0%9F%90%AE&createdTime=1698882012944)